ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -608,30 +608,35 @@ Loader {
                 }
 
                 replyDetails: StatusMessageDetails {
+                    readonly property var responseMessage: root.messageStore.getMessageByIdAsJson(responseToMessageWithId)
```

Comment:
If possible, add a condition to this to only fetch the `responseMessage` message when needed, so in this case when it's an image or a sticker.

Reply:
okay :)

Reply:
@jrainville fixed